backend/src/services/group/group.service.ts

```diff
@@ -47,9 +49,27 @@ class Group {
       permissionsIds,
     });
 
-    const newGroup = await this.#groupRepository.create(group);
+    return this.#groupRepository.create(group);
+  }
+
+  public async delete({ id }: EAMGroupDeleteParamsDto): Promise<number> {
```

Comment:
If you want to return something except from `void` from service you need to use a dto.
```suggestion
  public async delete({ id }: EAMGroupDeleteParamsDto): Promise<void> {
```